CHANGELOG.md

```diff
@@ -16,6 +16,12 @@
 
 ### Master
 
+- Fair exhibitors overview looks up to spec - orta
+- Hides additional info when a fair doesn't have it - orta
+- When there's only a website for the fair additional info, the app goes there - orta
+
+### 1.8.3
```

Comment:
This got shipped today, no idea about the boundaries for everyone else work, but mine is all post-1.8.3